src/include/network/lan/cmdline.rb

```diff
@@ -244,56 +243,37 @@ def EditHandler(options)
       end
       LanItems.SetItem(builder: builder)
 
-      if Builtins.size(Ops.get_string(LanItems.getCurrentItem, "ifcfg", "")) == 0
-        NetworkInterfaces.Add
-        LanItems.operation = :edit
```

Comment:
sadly removal of this does not work, as builder expect that Add and operation settings. ( I planned to add it there, but in the end do not do it )

Reply:
I was checking if was needed at all. I was able to edit a non configured one. So, was not sure.

Reply:
```bash
$ yast2 lan edit id=0 ip=192.168.0.4 netmask=255.255.255.0
## THE OVERVIEW IS WRONG
RTL-8100/8101L/8139 PCI Fast Ethernet Adapter (Not connected)
MAC : 52:54:00:6e:7c:0e
BusID : 0000:00:0b.0
Device Name: ens11
The device is not configured. Press Edit

$ yast2 lan show id=0
RTL-8100/8101L/8139 PCI Fast Ethernet Adapter (Not connected)
MAC : 52:54:00:6e:7c:0e
BusID : 0000:00:0b.0
Device Name: ens11
Started automatically at boot
IP address: 192.168.0.4/24
to configure.

$ cat /etc/sysconfig/network/ifcfg-ens11
BOND_SLAVES='[]' ## This is incorrect.
BOOTPROTO='static'
BROADCAST=''
ETHTOOL_OPTIONS=''
IPADDR='192.168.0.4/24'
MTU=''
NAME=''
NETWORK=''
PREFIX=''
PREFIXLEN=''
REMOTE_IPADDR=''
STARTMODE='auto'

Reply:
try to add completelly new device like new bridge. This is where add is needed.

Reply:
But then it is a completely different handler, isn't it? This is the edit handler, where the interface should be listed as configured or unconfigured.